src/compiler/emitter.ts

```diff
@@ -2468,6 +2468,21 @@ namespace ts {
         }
 
         function emitTripleSlashDirectives(files: ReadonlyArray<FileReference>, types: ReadonlyArray<FileReference>) {
+            if (currentSourceFile && currentSourceFile.moduleName) {
+                write(`/// <amd-module name="${currentSourceFile.moduleName}" />`);

```

Comment:
`/// <reference no-default-lib="true"/>` is the only one left out here.. well in addition to `/** @jsx */`